wrappers/python/pyrs_sensor.cpp

```diff
@@ -92,8 +92,10 @@ void init_sensor(py::module &m) {
             ss << ">";
             return ss.str();
         } );
-    // rs2::sensor_from_frame [frame.def("get_sensor", ...)?
-    // rs2::sensor==sensor?
+        m.def("sensor_from_frame", [](rs2::frame f) {
```

Comment:
It would be nice if the syntax was `rs.sensor.from_frame(f)` (a static function inside the python "sensor") -- easy change, I think?